cluster-scope/overlays/nerc-ocp-prod/externalsecrets/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 resources:
 - default-ingress-certificate.yaml
 - default-api-certificate.yaml
+- external-apps-ingress-certificate.yaml
```

Comment:
That file references:

```
  dataFrom:
    - extract:
        key: nerc/nerc-ocp-prod/openshift-ingress/external-apps-ingress-certificate
```

But no such key exists in the vault:

```
$ vault kv  list nerc/nerc-ocp-prod/openshift-ingress/
Keys
----
default-ingress-certificate
route53-creds
```

I'm checking to see if we have an existing certificate.

With access via the public network now working, we should be able to configure a letsencrypt-based secretstore for acquiring public certificates.

Reply:
@larsks Ah okay, that would make this very unhelpful in that case. All of this must have been set up as more of a layout than a working implementation. Let me know what we need here and if that cert exists